omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -711,16 +711,11 @@
             NPM=[NPM]
             EC2_USE_WINDOWS_PREFIX_DETECTION=[EC2_USE_WINDOWS_PREFIX_DETECTION]
             OVERRIDE_INSTALLATION_METHOD=[OVERRIDE_INSTALLATION_METHOD]
-            ' />
-    <CustomAction Id='FinalizeInstall' BinaryKey='wixcadll' DllEntry='FinalizeInstall' Execute='deferred' Return='check' Impersonate='no'/>
+            ' HideTarget='yes' />
+    <CustomAction Id='FinalizeInstall' BinaryKey='wixcadll' DllEntry='FinalizeInstall' Execute='deferred' Return='check' HideTarget='yes' Impersonate='no'/>
     <CustomAction Id='StopDDServices' BinaryKey='wixcadll' DllEntry='PreStopServices' Execute='deferred' Return='check' Impersonate='no'/>
     <CustomAction Id='StartDDServices' BinaryKey='wixcadll' DllEntry='PostStartServices' Execute='deferred' Return='ignore' Impersonate='no'/>
-
-    <CustomAction Id='SetUninstallProperty' Return='check' Property='DoUninstall'
```

Comment:
need to remove [declaration](https://github.com/DataDog/datadog-agent/pull/13442/files#diff-d6158508805a99eaed5c28596a83246becbca20978d481d7d87d14cc754e128aR477) too